src/Layers/xrRender/stats_manager.cpp

```diff
@@ -258,4 +270,4 @@ u32 get_format_pixel_size(DXGI_FORMAT format)
         // Do not consider extraordinary formats.
         return 0;
 }
-#endif
+#endif
```

Comment:
Don't remove whitespace in the end of file
```suggestion
#endif

```